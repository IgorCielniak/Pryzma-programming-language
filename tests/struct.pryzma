struct Person {
    name
    age = 30
}

p = Person {"Some, name", 30}

print p.name, "\n"
print p.age, "\n"

p2 = Person {}

p2.name = "Alice"

print p2
