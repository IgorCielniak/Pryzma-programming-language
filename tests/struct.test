python Pryzma.py tests/struct.pryzma
