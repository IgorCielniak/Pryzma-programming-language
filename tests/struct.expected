Some, name
30
{'name': 'Alice', 'age': 30}